packages/xrpl/src/models/transactions/accountSet.ts

```diff
@@ -155,10 +158,19 @@ const MAX_TICK_SIZE = 15
  * @param tx - An AccountSet Transaction.
  * @throws When the AccountSet is Malformed.
  */
-// eslint-disable-next-line max-lines-per-function -- okay for this method, only a little over
+// eslint-disable-next-line max-lines-per-function, max-statements -- okay for this method, only a little over
 export function validateAccountSet(tx: Record<string, unknown>): void {
   validateBaseTransaction(tx)
 
+  if (
+    tx.NFTokenMinter !== undefined &&
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment -- Necessary for isValidClassicAddress ts-expect-error below
+    // @ts-expect-error
+    !isValidClassicAddress(tx.NFTokenMinter)
```

Comment:
This avoids using `@ts-expect-error`

```suggestion
    !isValidClassicAddress(String(tx.NFTokenMinter))
```

Reply:
Fixed, thanks didn't knew this :)